sources.go

```diff
@@ -1,12 +1,13 @@
 package main
 
 import (
+	"bytes"
 	"fmt"
+	"github.com/klauspost/cpuid"
 	"io/ioutil"
 	"os/exec"
 	"path"
-
-	"github.com/klauspost/cpuid"
+	"strconv"
```

Comment:
I saw [this](http://goinbigdata.com/goimports-vs-gofmt/) article that indicates goimports is a superset of gofmt, it groups and orders the imports correctly in addition to format correction. I'll use goimports if it seems okay to you both?